modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -120,12 +120,18 @@ definition {
 	}
 
 	@description = "Verify that the user can add Nodes into diagram by drag&drop: Join XOR"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanAddJoinXorNode {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanAddNotification";
 
-		// TODO CanAddJoinXorNode pending implementation
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Title");
 
+		ProcessBuilderKaleoDesignerReact.dragSidebarNodeAndMove(
+			nodeLabel = "Join XOR",
+			nodePosition = "-40,-200",
+			nodeType = "join-xor");
 	}
```

Comment:
It would be interesting to add a `viewPanelNodePresent` to check the node in the diagram.

Reply:
Is there still a possibility to change? Yes, I missed that detail.
anything I can go back and put this step to the test to be more complete, ok